src/Server/cacheClient.tsx

```diff
@@ -85,24 +94,11 @@ const redisCacheSetup = cb => {
     return await safeCacheCommand("get", key)
   }
 
-  cache = {
+  return {
     get: cacheGet,
     set: cacheSet,
+    redisClient,
   }
-
-  cb()
 }
 
-export { cache }
-
-/**
- * Connect to Redis
- * TODO: Should we still hold off the server spin up until after the cache is loaded?
- */
-export function initializeCache() {
-  console.log("[Force] Initializing cache...")
-
-  return new Promise(resolve => {
-    artsyCache.setup(() => redisCacheSetup(resolve))
```

Comment:
For some reason we were initializing two nearly identical cache clients 🤷 Deleted the cameron one